cpu/cpupower.py

```diff
@@ -33,7 +33,7 @@ class cpupower(Test):
 
     def setUp(self):
         if not os.path.exists('/sys/devices/system/cpu/cpu0/cpufreq'):
-            self.cancel('sysfs directory for cpufreq is unavailable.')
+            self.cancel('cpufreq is supported only on PowerNV machines')
```

Comment:
The check is for directory existence, and the cancel message is saying about PowerNV.
Change the condition also please.